### PR TITLE
configurable ui docs base url #179

### PR DESCRIPTION
**Pull request by syncush**
feat(UI) : predator docs base URL is configurable via env variable if no such env exists fallbacks to https://zooz.github.io/predator

### CODE REVIEW

**Comment by NivLipetz** [comment]
#180

File: ui/package-lock.json [hunk 1]
```diff
@@ -5377,7 +5377,8 @@
         "ansi-regex": {
           "version": "2.1.1",
```
Comment (manorlh) [inline ui/package-lock.json]: - revert package-lock.json

File: ui/config/webpack.config.js [hunk 2]
```diff
@@ -29,6 +29,6 @@ module.exports = {
       BUCKET_PATH: env.BUCKET_PATH || '/',
     }),
     new webpack.NamedModulesPlugin(),
```
Comment (manorlh) [inline ui/config/webpack.config.js]: - Add the feature to README.md
- Add the new ARG to Dockerfile.  (like BUCKET_PATH)

**Comment by syncush** [comment]
Does it mean that one should build a new Dockerfile that pulls the Predator image and specify PREDATOR_DOCS_URL value?

Is it possible to determine the value at container startup?

**Comment by manorlh** [comment]
Hi
It is possible but no. 
The UI building taking time - something like 20 sec, so we do it at docker build and not docker run. we don't want that docker run will take a lot of time.

so, yeah you should build the docker with your's values on your own and then run it.

**Comment by syncush** [comment]
Okay, I will make the specified changes

**Comment by manorlh** [comment]
I opened a related issue that could help
https://github.com/Zooz/predator/issues/199

File: ui/package-lock.json (hunk 1 above)
Comment (manorlh) [reply ui/package-lock.json]: just that and we can merge it

**Comment by syncush** [comment]
@manorlh 
Aah, I think my Prettier made unwanted changes in the README file, I am on it

**Comment by syncush** [comment]
Close #180

**Pull request merged**